resources/contributors.txt

```diff
@@ -43,6 +43,7 @@ matthewde | https://github.com/m4tthewde | :/avatars/matthewde.jpg | Contributor
 Karar Al-Remahy | https://github.com/KararTY | :/avatars/kararty.png | Contributor
 Talen | https://github.com/talneoran | | Contributor
 SLCH | https://github.com/SLCH | :/avatars/slch.png | Contributor
+xHeaveny | https://github.com/xHeaveny | :/avatars/xheaveny.png | Contributor
```

Comment:
Adding yourself to contributors should be a separate PR.

Reply:
So should I take it out?

Reply:
> So should I take it out?

Yes

Reply:
Done